philips-hue-adapter.js

```diff
@@ -365,8 +380,21 @@ class PhilipsHueDevice extends Device {
                     type: 'boolean',
                     readOnly: true,
                   },
-                  device.state.buttonevent >= buttonInfo.start &&
-                                       device.state.buttonevent < buttonInfo.off
+                  device.state.buttonevent >= buttonInfo.initialPress &&
+                    device.state.buttonevent <= buttonInfo.longRelease
+                )
+              );
+              this.properties.set(
+                'lastUpdated',
+                new PhilipsHueProperty(
+                  this,
+                  buttonType,
```

Comment:
This should be `'lastUpdated'`, not `buttonType`.

Reply:
Done